pages/common/git-merge-base.md

```diff
@@ -0,0 +1,16 @@
+# git merge-base
+
+> Find a common ancestor of two commits.
+> More information: <https://git-scm.com/docs/git-merge-base>.
+
+- Output a best common ancestor of two commits:
+
+`git merge-base {{commit_1}} {{commit_2}}`
+
+- Output all best common ancestors of two commits:
+
+`git merge-base --all {{commit_1}} {{commit_2}}`
```

Comment:
```suggestion
`git merge-base --all {{commit1}} {{commit2}}`
```

Reply:
Why not commit_hash instead

Reply:
I think the git subcommand pages are not consistent in this matter, but the majority uses just commit.

Reply:
> I think the git subcommand pages are not consistent in this matter, but the majority uses just commit.

It means that consistency should be fixed. For instance we can write script to do that like in #9578 PR.